tests/utils/nodes/nodes.go

```diff
@@ -93,3 +93,16 @@ func setUnSchedulableValue(clients *client.ClientSet, nodeName string, unSchedul
 
 	return nil
 }
+
+func IsNodeMaster(name string, clients *client.ClientSet) (bool, error) {
+	node, err := clients.Nodes().Get(context.Background(), name, metav1.GetOptions{})
+	if err != nil {
+		return false, err
+	}
+
+	if _, exists := node.Labels["node-role.kubernetes.io/master"]; exists {
```

Comment:
I think you might have to handle `control-plane` as well. 

Similar to: https://github.com/test-network-function/cnf-certification-test/pull/512

Reply:
Also: https://github.com/test-network-function/cnf-certification-test/blob/main/pkg/provider/provider.go#L62 for reference.

Reply:
Good catch!

Reply:
Thanks ! fixed